### PR TITLE
[TASK][27022] create models, migrations, seeds

### PR DESCRIPTION
**Pull request by chinhhq-0603**
https://edu-redmine.sun-asterisk.vn/issues/27022

### CODE REVIEW

File: spec/models/warehouse_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```
Comment (chienpt98) [inline spec/models/warehouse_spec.rb]: nháy " "

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```
Comment (chienpt98) [inline spec/models/user_spec.rb]: nháy " "

File: spec/models/plane_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```
Comment (chienpt98) [inline spec/models/plane_spec.rb]: " "

File: spec/models/location_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```
Comment (chienpt98) [inline spec/models/location_spec.rb]: " "

File: spec/models/history_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```
Comment (chienpt98) [inline spec/models/history_spec.rb]: " "

File: spec/models/component_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
```
Comment (chienpt98) [inline spec/models/component_spec.rb]: " "

File: db/migrate/20200814025037_create_users.rb [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+      t.datetime :activated_at
+      t.boolean :activated, default: false
+      t.string :reset_digest
+      t.datetime :reset_sent_at
+      t.references :warehouse, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```
Comment (chienpt98) [inline db/migrate/20200814025037_create_users.rb]: endline

**Review by chienpt98 — APPROVED** [review]

**Comment by chinhhq-0603** [comment]
ready

**Review by hieplt-0914 — COMMENTED** [review]
コンベンションを修理してください。

File: app/models/user.rb [hunk 8]
```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  belongs_to :warehouse
```
Comment (hieplt-0914) [inline app/models/user.rb:2]: 1. Hằng số
2. Attribute accessor/reader/writer
3. Association
4. Nested attributes
5. Deligate
6. Validates
7. Callback
8. Scope
9. Public method
10. Private method

Comment (chinhhq-0603) [reply app/models/user.rb:2]: ok a

**Comment by chinhhq-0603** [comment]
ready

**Pull request merged**